src/tools/tidy/src/main.rs

```diff
@@ -69,6 +69,9 @@ fn main() {
         check!(errors, &compiler_path);
         check!(error_codes_check, &[&src_path, &compiler_path]);
 
+        // Checks for rustdoc.
+        check!(rustdoc_html_ids, &src_path);
```

Comment:
Could you add a new `rustdoc_path` variable that is defined like this? We shouldn't be checking all the other folders in `src/`, especially because it'll make the check much slower.

```
let rustdoc_path = root_path.join("src/librustdoc");
```

Reply:
Sure!